src/Workspaces/Core/Portable/Workspace/Solution/DocumentState.cs

```diff
@@ -603,44 +580,18 @@ private VersionStamp GetNewTreeVersionForUpdatedTree(SyntaxNode newRoot, Version
             SourceHashAlgorithm checksumAlgorithm,
             DocumentInfo.DocumentAttributes attributes,
             ParseOptions options,
-            ISyntaxTreeFactoryService factory,
-            PreservationMode mode)
+            ISyntaxTreeFactoryService factory)
         {
```

Comment:
Does this method need a rename, since neither the tree nor text are recoverable anymore? Or can it just be inlined?

Reply:
will do in followup!

Reply:
done in followup.